lib/tracker/input.go

```diff
@@ -75,9 +74,10 @@ func WithPruneTiming(pruneTick, pruneAfter time.Duration) Option {
 		t.pruneAfter = pruneAfter
 	}
 }
-func WithPrometheusCounters(currentPlanes prometheus.Gauge) Option {
+func WithPrometheusCounters(currentPlanes prometheus.Gauge, decodedFrames prometheus.Counter) Option {
```

Comment:
Which other metric would we use to determine the % of decoded frames? I'm just thinking of what would make for a useful metric to track - especially between versions as we hone the decoding logic.

Reply:
we could the total incoming frames and the deduped frames i think